src/log-service/templates/config.yaml

```diff
@@ -15,6 +15,9 @@ data:
   LOG_SERVICE_DISABLE_AUTH: "true"
   LOG_SERVICE_GLOBAL_TOKEN: c76e567a-b341-404d-a8dd-d9738714eb82
   LOG_SERVICE_SECRET: IC04LYMBf1lDP5oeY4hupxd4HJhLmN6azUku3xEbeE3SUx5G3ZYzhbiwVtK4i7AmqyU9OZkwB4v8E9qM
+  LOG_SERVICE_REDIS_MASTER_NAME: "harness-redis"
```

Comment:
I have a doubt, for vm based redis instance. Will it be using sentinels?

Reply:
nope, single server, master name is only required for in-cluster sentinels